### DISCUSSION TITLE
[11일차-윤경]http와 https의 차이점

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#39] [프론트 면접 질문]
http와 https의 차이점은 무엇인가요?

**Comment by skyhanull** [comment]
HTTP(Hyper Text Transfer Protocol)는 서버/클라이언트 모델을 따라 데이터를 주고 받기 위한 프로토콜입니다
포트는 80번 포트를 사용하며 보안이 적용되지 않는 데이터를 전송합니다
HTTPS는 위의 HTTP에 데이터 암호화가 추가된 프로토콜을 의미합니다 포트는 443번 포트를 사용합니다
즉 HTTP를 SSL(Secure Sockets Layer) 프로토콜 위에서 돌아가도록하여 클라이언트와 서버가 주고받는 텍스트를 암호화합니다.

**Comment by ez1n** [comment]
- **http**
HyperText Transfer Protocol의 약자
인터넷을 작동시키는 역할을 하며, 웹 서버 및 웹 브라우저 상호 간의 데이터 전송을 위한 응용계층프로토콜이다.
HTTP 프로토콜은 서버에서부터 브라우저로 전송되는 정보가 암호화되지 않아 보안면에서 취약하다는 단점이 있다.

- **https**
HyperText Transfer Protocol Secure의 약자
표준 HTTP와 동일한 방식으로 작동하지만 SSL(보안 소켓 계층)을 사용함으로써 서버와 주고받는 데이터가 암호화되기 때문에 서버와 브라우저가 민감한 정보를 주고받을 때 도난당하는 것을 방지한다.

**Comment by smarfy99** [comment]
http
- hypertext transfer protocol 로 웹 서버 및 웹 브라우저 상호 간의 데이터 전송을 위한 응용계층 프로토콜

https
- http 프로토콜에 SSL(secure socket layer) 보안 인증서를 사용한 보안용 프로토콜이다.
